### PR TITLE
MP JWT 1.2 blog: Updates from review

### PR DESCRIPTION
**Pull request by ayoho**
For #1168

### CODE REVIEW

File: posts/2021-03-24-whats-new-in-MP-JWT-1.2.adoc [hunk 1]
```diff
@@ -1,21 +1,21 @@
 ---
 layout: post
-title: "MicroProfile JSON Web Token 1.2"
+title: "MicroProfile JWT RBAC 1.2"
```
Comment (tevans78) [inline posts/2021-03-24-whats-new-in-MP-JWT-1.2.adoc]:
```suggestion
title: "MicroProfile JWT 1.2"
```

File: posts/2021-03-24-whats-new-in-MP-JWT-1.2.adoc [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 categories: blog
 author_picture: https://avatars3.githubusercontent.com/ayoho
 author_github: https://github.com/ayoho
-seo-title: MicroProfile JSON Web Token 1.2 - OpenLiberty.io
-seo-description: This post demonstrates how to configure an Open Liberty application with the latest enhancements to the MicroProfile JSON Web Token feature. New capabilities include support for new MicroProfile Config properties, JSON Web Encryption (JWE) tokens, and enhanced algorithm support.
-blog_description: "This post demonstrates how to configure an Open Liberty application with the latest enhancements to the MicroProfile JSON Web Token feature. New capabilities include support for new MicroProfile Config properties, JSON Web Encryption (JWE) tokens, and enhanced algorithm support."
+seo-title: MicroProfile JWT RBAC 1.2 - OpenLiberty.io
```
Comment (tevans78) [inline posts/2021-03-24-whats-new-in-MP-JWT-1.2.adoc]:
```suggestion
seo-title: MicroProfile JWT 1.2 - OpenLiberty.io
```

File: posts/2021-03-24-whats-new-in-MP-JWT-1.2.adoc [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+seo-description: This post demonstrates how to configure an Open Liberty application with the latest enhancements to the MicroProfile JWT feature. New capabilities include support for new MicroProfile Config properties, JSON Web Encryption (JWE) tokens, and enhanced algorithm support.
+blog_description: "This post demonstrates how to configure an Open Liberty application with the latest enhancements to the MicroProfile JWT feature. New capabilities include support for new MicroProfile Config properties, JSON Web Encryption (JWE) tokens, and enhanced algorithm support."
 open-graph-image: https://openliberty.io/img/twitter_card.jpg
 ---
-= MicroProfile JSON Web Token 1.2
+= MicroProfile JWT RBAC 1.2
```
Comment (tevans78) [inline posts/2021-03-24-whats-new-in-MP-JWT-1.2.adoc]:
```suggestion
= MicroProfile JWT 1.2
```

File: posts/2021-03-24-whats-new-in-MP-JWT-1.2.adoc [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
 Adam Yoho <https://github.com/ayoho>
 :imagesdir: /
 :url-prefix:
 :url-about: /
 
-The MicroProfile JSON Web Token specification allows using a JWT for authenticating and authorizing requests to a service. The latest version of the specification simplifies the configuration for managing the validation of the JWT by introducing new MicroProfile Config properties. Support for JSON Web Encryption (JWE) tokens and new signature algorithms are also added in this feature.
+The link:https://github.com/eclipse/microprofile-jwt-auth/releases/tag/1.2[MicroProfile JWT RBAC 1.2 specification] enables use a JSON Web Token for authenticating and authorizing requests to a service. The latest version of the specification simplifies the configuration for managing the validation of the JWT by introducing new MicroProfile Config properties. Support for JSON Web Encryption (JWE) tokens and new signature algorithms are also added in this feature.
```
Comment (tevans78) [inline posts/2021-03-24-whats-new-in-MP-JWT-1.2.adoc]:
```suggestion
The link:https://github.com/eclipse/microprofile-jwt-auth/releases/tag/1.2[MicroProfile JWT RBAC 1.2 specification] enables use of a JSON Web Token for authenticating and authorizing requests to a service. The latest version of the specification simplifies the configuration for managing the validation of a JWT by introducing new MicroProfile Config properties. Support for JSON Web Encryption (JWE) tokens and new signature algorithms are also added in this feature.
```

**Comment by ayoho** [comment]
Closing in favor of https://github.com/OpenLiberty/blogs/pull/1198

**Pull request closed without merging**